numpy/core/src/private/ufunc_override.c

```diff
@@ -8,148 +8,172 @@
 
 #include "ufunc_override.h"
 
-static void
+
+static int
 normalize___call___args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds,
-                    int nin)
+                        PyObject **normal_args, PyObject **normal_kwds,
+                        int nin, int nout)
```

Comment:
Not as long as we're passing `np.dot` and `np.matmul` through here... I think I should start collecting to-do clean-up items...